modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -363,105 +326,133 @@ export default class SelectBase extends React.Component<SelectBaseProps> {
-      onMenuCloseOnEscape,
-      options,
-      required,
-      shouldMenuAnimate,
-      shouldMenuAutoFlip,
-      shouldMenuAutoFocus,
-      value,
-
-      // Strip unneeded props on button from elemProps
-      onOptionSelection,
-
-      ...elemProps
-    } = this.props;
-
-    // Use default renderOption if renderOption prop isn't provided
-    const renderOption = this.props.renderOption || this.renderOption;
-
-    // Do a bit of error-checking in case options weren't provided
-    const selectedOption = !isEmpty ? options[getCorrectedIndexByValue(options, value)] : null;
-    const selectedOptionLabel = selectedOption ? selectedOption.label : '';
-    const selectedOptionValue = selectedOption ? selectedOption.value : '';
-
-    return (
-      <SelectWrapper grow={grow} disabled={disabled}>
-        <SelectButton
-          aria-expanded={!isMenuHidden ? 'true' : undefined}
-          aria-haspopup="listbox"
-          aria-controls={!isMenuHidden ? this.menuId : undefined}
-          disabled={disabled}
+  // If the focused option changed, scroll the newly focused option into view (if
+  // necessary) but do NOT center it
+  useLayoutEffect(() => {
+    const focusedOption = focusedOptionRef.current;
+
+    if (focusedOption) {
+      // TODO: Figure out why rAF is necessary here (and below) in order for the
+      // Select States Menu On story to render correctly in IE. Without rAF,
+      // the DOM measurements performed in scrollIntoViewIfNeeded are incorrect
+      // and we perform unnecessary scrolling BUT ONLY IN THIS STORY and ONLY IN
+      // IE. Everywhere else (in IE and in other browsers), everything works
+      // fine without rAF.
+      const animateId = requestAnimationFrame(() => {
+        // We cannot use the native Element.scrollIntoView() here because it
+        // doesn't work properly with the portalled menu: when using the keyboard
+        // to advance focus through the options, using scrollIntoView to keep the
+        // newly focused option in view also scrolls the ENTIRE page. Instead, we
+        // call our own scrollIntoViewIfNeeded function.
+        scrollIntoViewIfNeeded(focusedOption, false);
+      });
+
+      return () => {
+        cancelAnimationFrame(animateId);
+      };
+    }
+
+    return undefined;
+  }, [focusedOptionIndex]);
+
+  // If the menu was just opened, scroll the focused option into view (if
+  // necessary) and center it
+  useLayoutEffect(() => {
+    const focusedOption = focusedOptionRef.current;
+
+    // We need to scroll if the menu is either opening or opened in case we decide to
+    // bypass the opening state and jump straight to opened (like in visual testing,
+    // for instance)
+    if (focusedOption && (menuVisibility === 'opening' || menuVisibility === 'opened')) {
+      // TODO: Again, why is rAF necessary here in IE? (see above)
+      const animateId = requestAnimationFrame(() => {
+        scrollIntoViewIfNeeded(focusedOption, true);
+      });
```

Comment:
This `rAF` call needs to be present as well in order for `Select States Menu On` to be rendered properly in IE (**both** `rAF` calls need to be present).

Reply:
Again, added a bit of extra info in the code comments for this rAF call. Resolving this and adding an updated comment.